src/test-run/commands/index.js

```diff
@@ -102,6 +126,9 @@ export function createCommandFromObject (obj) {
     if (obj.type === TYPE.doubleClick)
         return new DoubleClickCommand(obj);
 
+    if (obj.type === TYPE.hover)
+        return new HoverCommand(obj);
+
     if (obj.type === TYPE.testDone)
         return new TestDoneCommand();
 }
```

Comment:
Also take a look here, it's very similar to conditional statements in `execute-action-command.js`.